### PR TITLE
add custom predictor example : gpu parallel inference

### PR DESCRIPTION
**Pull request by park12sj**
## Proposed Changes 

- I added it because I thought it would be good to have an example of applying parallel inference to gpu.

### CODE REVIEW

File: docs/modelserving/v1beta1/custom/custom_model/README.md [hunk 1]
```diff
@@ -75,6 +75,25 @@ class AlexNetModel(kserve.Model):
 if __name__ == "__main__":
     kserve.ModelServer().start({"custom-model": AlexNetModel})
 ```
+gpu example
+```python
+@serve.deployment(name="custom-model", num_replicas=2, ray_actor_options={"num_cpus":1, "num_gpus": 0.5})
```
Comment (yuzisun) [inline docs/modelserving/v1beta1/custom/custom_model/README.md:80]: Does Ray support fractional gpu ?

Comment (park12sj) [reply docs/modelserving/v1beta1/custom/custom_model/README.md:80]: @yuzisun 
Yes, ray supports fractional gpu.
https://docs.ray.io/en/latest/serve/scaling-and-resource-allocation.html#serve-omp-num-threads
image

Comment (yuzisun) [reply docs/modelserving/v1beta1/custom/custom_model/README.md:80]: @park12sj Sorry for the late reply, can you help add this reference so people can read more details from Ray website?

File: docs/modelserving/v1beta1/custom/custom_model/README.md [hunk 2]
```diff
@@ -75,6 +75,27 @@ class AlexNetModel(kserve.Model):
...
+        pass
+
+    def predict(self, request: Dict) -> Dict:
+        pass
+
+if __name__ == "__main__":
+    ray.init(num_cpus=2, num_gpus=1)
+    kserve.ModelServer().start({"custom-model": AlexNetModel})
+```
+The more details for ray fractional cpu and gpu can be found [here](https://docs.ray.io/en/latest/serve/scaling-and-resource-allocation.html#fractional-cpus-and-fractional-gpus).
```
Comment (park12sj) [inline docs/modelserving/v1beta1/custom/custom_model/README.md:97]: @yuzisun 
Would it be okay if I added it like this?

**Comment by yuzisun** [comment]
Thanks @park12sj !

/lgtm
/approve

**Pull request merged**